classes/item.rb

```diff
@@ -0,0 +1,27 @@
+require 'date'
+
+class Item
+  attr_reader :publish_date
+  attr_accessor :archived
+
+  def initialize(genre, author, source, label, publish_date)
```

Comment:
- [ ] Please Add a setter and getter for the (genre, author, source ...etc) to make accessible.